index.html

```diff
@@ -6,35 +6,187 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>@Sir_Malcom</title>
     <link rel="stylesheet" href="style.css">
+    <style>@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');</style>
+    <script src="https://kit.fontawesome.com/b5d9cbac6d.js" crossorigin="anonymous"></script>
 </head>
 <body>
   <header>
-    <img class="logo" src="./elements/lg.png" alt="Logo">
+    <a href="#"><p class="logo">MY LOGO</p></a>
     <nav class="dcp1 navbar">
-        <img class="menu" src="./elements/Enabled.jpg" alt="Humbugger menu">
+      <img class="menu" src="./elements/Enabled.jpg" alt="Humbugger menu">
     </nav>
   </header>
-  <section class="parent1">   
-      <div class="space"></div>
-      <section class="dcp1 art1">
-          <h1>
-        I’m Sir_Malcom<br>Glad to see<br>you!
-          </h1>
-        </section>
-      <p class="dcp1 art2">
-        I’m a software developer! I can help you build a product, 
-        feature or website Look through some of my work and experience! 
-        If you like what you see and have a project you need coded, 
-        don’t hesitate to contact me.
-      </p>
-      <img class="dcp1 social" src="./elements/ltscnnct.jpg" alt="lets connect">
-      <ul class="dcp1 parent2">
-        <li><a href="#" target="_blank"><img src="./elements/gthb.png" alt="github"></a></li>
-        <li><a href="#" target="_blank"><img src="./elements/lnkin.png" alt="linkedin"></a></li>
-        <li><a href="#" target="_blank"><img src="./elements/twtr.png" alt="twitter"></a></li>
-        <li><a href="#" target="_blank"><img src="./elements/m.png" alt="m"></a></li>
-        <li><a href="#" target="_blank"><img src="./elements/chill.png" alt="chill"></a></li>
-        </ul>
+  <section class="intro">   
+    <div class="space"></div>
+      <h1 class="dcp1 art1">
+      I’m Sir_Malcom<br>Glad to see<br>you!
```

Comment:
[OPTIONAL]

I highly encourage you to use your real name instead of a surename, it will definetly make your portfolio seem more professional 💼